tools/ci/test-jobs.yml

```diff
@@ -0,0 +1,187 @@
+# Job configurations for `.github/workflows/test.yml`
+#
+# Splitting these out into a separate file seems to be the only sane way to
+# conditionally filter out jobs that should only be run on cron; cf.
+# <https://stackoverflow.com/q/65384420/744178>.
+
+- python-version: '3.7'
+  extra-envs: {}
+
+- python-version: '3.11'
+  # Run all tests in a single whoop here
+  # We cannot have empty -A selector, so the one which always will be fulfilled
+  extra-envs:
+    PYTEST_SELECTION: ""
+    PYTEST_SELECTION_OP: "not"
+    # To test https://github.com/datalad/datalad/pull/4342 fix.
+    # From our testing in that PR seems to have no effect, but kept around
+    # since should not hurt.
+    LC_ALL: ru_RU.UTF-8
```

Comment:
I wonder also if for this one we better `apt install locales` or even `apt install locales-all` (I locally do not hit segfault and do have `locales` but not `locales-all`)

Reply:
it seems to be it! installing locales-all addresses segfault.  I will wait for this test run to complete and then will clean it up ...